src/utils/getSymbolScale.ts

```diff
@@ -1,16 +1,24 @@
 import {scaleOrdinal} from 'd3-scale'
-import {SymbolType, symbols} from 'd3-shape'
 import {Table, Scale} from '../types'
 import {SYMBOL_COL_KEY} from '../constants'
 
+export enum SymbolType {
```

Comment:
Can you make this a union of string literals? I think string literals are friendlier to consumers of the project that don't use TypeScript.
```
export type Symbol = 'circle' | 'triangle' | 'square'
```
It should still provide the same type safety / benefits to us.